cupboard_app/views.py

```diff
@@ -784,3 +791,83 @@ def list(self, request: Request) -> Response:
         all_measurements = get_all_measurements()
         serializer = MeasurementSerializer(all_measurements, many=True)
         return Response(serializer.data, status=200)
+        return Response(serializer.data)
+
+
+@extend_schema(tags=['CustomIngredients'])
+class CustomIngredientsViewSet(viewsets.ViewSet):
+    MISSING_ING = 'Missing ingredient and type in message body.'
+
+    @extend_schema(
+        request=None,
```

Comment:
Add request body here instead of none. It should be something like:
```
       request = inline_serializer(
            name='AddCustomIngredientRequest',
            fields={
                'ingredient': serializers.CharField(),
                'type': serializers.CharField()
            }
       )
```